locales/id/main.json

```diff
@@ -1,7 +1,7 @@
 {
   "hero": {
-    "heading": "Mitra Anda dalam Membangun Perangkat Lunak yang Andal dan Modern",
-    "desc": "Kami membantu organisasi menghadirkan sistem digital yang skalabel dan berkualitas tinggi melalui keunggulan rekayasa dan teknologi cloud-native."
+    "heading": "Mitra anda dalam membangun perangkat lunak yang andal dan modern",
```

Comment:
Please change it to: "Mitra Anda dalam membangun perangkat lunak yang andal dan modern"